### DISCUSSION TITLE
[ 功能建议 ]建议加一个sql熔断的功能

### DISCUSSION THREAD

**Discussion by czxin788** [hhyo/Archery#1583] [Ideas]
### 目前遇到的问题/使用障碍

 数据库出现cpu高时，在允许的情况下，通过archery底层用pt-kill工具，按照某个维度（用户、SQL等）进行在线杀sql，先让数据库cpu降下来，保证业务可用。
然后再慢慢分析原因。

**Comment by nick2wang** [comment]
会话管理结合搜索，应该能满足一部分需求
